scripts/merge-measures-data.js

```diff
@@ -114,5 +114,29 @@ function mergeQpp(qppJson) {
 
   console.error('did not find measure details for the following', measuresNotFound);
 
+  enrichACIMeasures(qppJson);
+
   return JSON.stringify(qppJson, null, 2);
 }
+
+
+/**
+ * Will add extra metadata to ACI measures
```

Comment:
Can you add a little more color here? What kind of metadata, where does the metadata come from (how is aci-measure-relations generated)?